src/GroupTypeManager.php

```diff
@@ -191,8 +191,22 @@ public function getGroupBundleIdsByEntityType($entity_type_id) {
    * {@inheritdoc}
    */
   public function getAllGroupBundles($entity_type = NULL) {
+    // Todo - should be remove since this method don't do any thing.
+    return $this->getGroupMap();
+  }
+
+  /**
+   * Get group bundles of an entity type.
+   *
+   * @param string $entity_type_id
+   *   The entity type ID.
+   *
+   * @return array
+   *   An associative array of bundle IDs, or an empty array if none found.
+   */
+  public function getGroupBundlesByEntityType($entity_type_id) {
```

Comment:
There is already a method for this: `getAllGroupBundles()`. I agree that this is a better name, but this is out of scope for this ticket. If we want to make this change let's do it in a dedicated PR and keep the focus of this PR on removing the complex widget.

Reply:
Also see https://github.com/Gizra/og/pull/181/files#r199885391

> Let's remove this method, it exactly duplicates the code from GroupTypeManager::getGroupsForEntityType().

Reply:
Reverted this change and fixed getAllGroupBundles in #402.